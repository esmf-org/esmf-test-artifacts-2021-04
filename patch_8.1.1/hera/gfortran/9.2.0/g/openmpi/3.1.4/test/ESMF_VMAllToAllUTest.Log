build time -- 2021-04-20 19:44:48
20210420 195134.038 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210420 195134.038 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210420 195134.038 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210420 195134.038 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210420 195134.038 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210420 195134.038 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210420 195134.038 INFO             PET0 Running with ESMF Version   : ESMF_8_1_1_beta_snapshot_02
20210420 195134.038 INFO             PET0 ESMF library build date/time: "Apr 20 2021" "19:43:22"
20210420 195134.038 INFO             PET0 ESMF library build location : /scratch1/NCEPDEV/da/Mark.Potts/sandbox/gfortran_9.2.0_openmpi_g_patch_8.1.1
20210420 195134.038 INFO             PET0 ESMF_COMM                   : openmpi
20210420 195134.039 INFO             PET0 ESMF_MOAB                   : enabled
20210420 195134.039 INFO             PET0 ESMF_LAPACK                 : enabled
20210420 195134.039 INFO             PET0 ESMF_NETCDF                 : enabled
20210420 195134.039 INFO             PET0 ESMF_PNETCDF                : disabled
20210420 195134.039 INFO             PET0 ESMF_PIO                    : enabled
20210420 195134.039 INFO             PET0 ESMF_YAMLCPP                : enabled
20210420 195134.039 INFO             PET0  Beginning Test, file ESMF_VMAllToAllUTest.F90, line 72
20210420 195134.039 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210420 195134.039 INFO             PET0  PASS  AllToAll Test iarray1 -> iarray2, ESMF_VMAllToAllUTest.F90, line 127
20210420 195134.039 INFO             PET0  PASS  Verify iarray2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 141
20210420 195134.039 INFO             PET0  PASS  AllToAll Test iarray2 -> iarray3, ESMF_VMAllToAllUTest.F90, line 151
20210420 195134.039 INFO             PET0  PASS  Verify iarray3 data against iarray1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 165
20210420 195134.039 INFO             PET0  PASS  AllToAll Test r4array1 -> r4array2, ESMF_VMAllToAllUTest.F90, line 177
20210420 195134.039 INFO             PET0  PASS  Verify r4array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 191
20210420 195134.039 INFO             PET0  PASS  AllToAll Test r4array2 -> r4array3, ESMF_VMAllToAllUTest.F90, line 201
20210420 195134.039 INFO             PET0  PASS  Verify r4array3 data against r4array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 215
20210420 195134.039 INFO             PET0  PASS  AllToAll Test r8array1 -> r8array2, ESMF_VMAllToAllUTest.F90, line 227
20210420 195134.039 INFO             PET0  PASS  Verify r8array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 241
20210420 195134.039 INFO             PET0  PASS  AllToAll Test r8array2 -> r8array3, ESMF_VMAllToAllUTest.F90, line 251
20210420 195134.039 INFO             PET0  PASS  Verify r8array3 data against r8array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 265
20210420 195134.039 INFO             PET0  Ending Test, file ESMF_VMAllToAllUTest.F90, line 267
20210420 195134.039 INFO             PET0 Finalizing ESMF
20210420 195134.038 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210420 195134.038 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210420 195134.038 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210420 195134.038 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210420 195134.038 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210420 195134.038 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210420 195134.038 INFO             PET1 Running with ESMF Version   : ESMF_8_1_1_beta_snapshot_02
20210420 195134.038 INFO             PET1 ESMF library build date/time: "Apr 20 2021" "19:43:22"
20210420 195134.038 INFO             PET1 ESMF library build location : /scratch1/NCEPDEV/da/Mark.Potts/sandbox/gfortran_9.2.0_openmpi_g_patch_8.1.1
20210420 195134.039 INFO             PET1 ESMF_COMM                   : openmpi
20210420 195134.039 INFO             PET1 ESMF_MOAB                   : enabled
20210420 195134.039 INFO             PET1 ESMF_LAPACK                 : enabled
20210420 195134.039 INFO             PET1 ESMF_NETCDF                 : enabled
20210420 195134.039 INFO             PET1 ESMF_PNETCDF                : disabled
20210420 195134.039 INFO             PET1 ESMF_PIO                    : enabled
20210420 195134.039 INFO             PET1 ESMF_YAMLCPP                : enabled
20210420 195134.039 INFO             PET1  Beginning Test, file ESMF_VMAllToAllUTest.F90, line 72
20210420 195134.039 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210420 195134.039 INFO             PET1  PASS  AllToAll Test iarray1 -> iarray2, ESMF_VMAllToAllUTest.F90, line 127
20210420 195134.039 INFO             PET1  PASS  Verify iarray2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 141
20210420 195134.039 INFO             PET1  PASS  AllToAll Test iarray2 -> iarray3, ESMF_VMAllToAllUTest.F90, line 151
20210420 195134.039 INFO             PET1  PASS  Verify iarray3 data against iarray1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 165
20210420 195134.039 INFO             PET1  PASS  AllToAll Test r4array1 -> r4array2, ESMF_VMAllToAllUTest.F90, line 177
20210420 195134.039 INFO             PET1  PASS  Verify r4array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 191
20210420 195134.039 INFO             PET1  PASS  AllToAll Test r4array2 -> r4array3, ESMF_VMAllToAllUTest.F90, line 201
20210420 195134.039 INFO             PET1  PASS  Verify r4array3 data against r4array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 215
20210420 195134.039 INFO             PET1  PASS  AllToAll Test r8array1 -> r8array2, ESMF_VMAllToAllUTest.F90, line 227
20210420 195134.039 INFO             PET1  PASS  Verify r8array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 241
20210420 195134.039 INFO             PET1  PASS  AllToAll Test r8array2 -> r8array3, ESMF_VMAllToAllUTest.F90, line 251
20210420 195134.039 INFO             PET1  PASS  Verify r8array3 data against r8array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 265
20210420 195134.039 INFO             PET1  Ending Test, file ESMF_VMAllToAllUTest.F90, line 267
20210420 195134.039 INFO             PET1 Finalizing ESMF
20210420 195134.038 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210420 195134.038 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210420 195134.038 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210420 195134.038 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210420 195134.038 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210420 195134.038 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210420 195134.038 INFO             PET2 Running with ESMF Version   : ESMF_8_1_1_beta_snapshot_02
20210420 195134.038 INFO             PET2 ESMF library build date/time: "Apr 20 2021" "19:43:22"
20210420 195134.038 INFO             PET2 ESMF library build location : /scratch1/NCEPDEV/da/Mark.Potts/sandbox/gfortran_9.2.0_openmpi_g_patch_8.1.1
20210420 195134.038 INFO             PET2 ESMF_COMM                   : openmpi
20210420 195134.039 INFO             PET2 ESMF_MOAB                   : enabled
20210420 195134.039 INFO             PET2 ESMF_LAPACK                 : enabled
20210420 195134.039 INFO             PET2 ESMF_NETCDF                 : enabled
20210420 195134.039 INFO             PET2 ESMF_PNETCDF                : disabled
20210420 195134.039 INFO             PET2 ESMF_PIO                    : enabled
20210420 195134.039 INFO             PET2 ESMF_YAMLCPP                : enabled
20210420 195134.039 INFO             PET2  Beginning Test, file ESMF_VMAllToAllUTest.F90, line 72
20210420 195134.039 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210420 195134.039 INFO             PET2  PASS  AllToAll Test iarray1 -> iarray2, ESMF_VMAllToAllUTest.F90, line 127
20210420 195134.039 INFO             PET2  PASS  Verify iarray2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 141
20210420 195134.039 INFO             PET2  PASS  AllToAll Test iarray2 -> iarray3, ESMF_VMAllToAllUTest.F90, line 151
20210420 195134.039 INFO             PET2  PASS  Verify iarray3 data against iarray1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 165
20210420 195134.039 INFO             PET2  PASS  AllToAll Test r4array1 -> r4array2, ESMF_VMAllToAllUTest.F90, line 177
20210420 195134.039 INFO             PET2  PASS  Verify r4array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 191
20210420 195134.039 INFO             PET2  PASS  AllToAll Test r4array2 -> r4array3, ESMF_VMAllToAllUTest.F90, line 201
20210420 195134.039 INFO             PET2  PASS  Verify r4array3 data against r4array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 215
20210420 195134.039 INFO             PET2  PASS  AllToAll Test r8array1 -> r8array2, ESMF_VMAllToAllUTest.F90, line 227
20210420 195134.039 INFO             PET2  PASS  Verify r8array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 241
20210420 195134.039 INFO             PET2  PASS  AllToAll Test r8array2 -> r8array3, ESMF_VMAllToAllUTest.F90, line 251
20210420 195134.039 INFO             PET2  PASS  Verify r8array3 data against r8array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 265
20210420 195134.039 INFO             PET2  Ending Test, file ESMF_VMAllToAllUTest.F90, line 267
20210420 195134.039 INFO             PET2 Finalizing ESMF
20210420 195134.038 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210420 195134.038 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210420 195134.038 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210420 195134.038 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210420 195134.038 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210420 195134.038 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210420 195134.038 INFO             PET3 Running with ESMF Version   : ESMF_8_1_1_beta_snapshot_02
20210420 195134.038 INFO             PET3 ESMF library build date/time: "Apr 20 2021" "19:43:22"
20210420 195134.038 INFO             PET3 ESMF library build location : /scratch1/NCEPDEV/da/Mark.Potts/sandbox/gfortran_9.2.0_openmpi_g_patch_8.1.1
20210420 195134.038 INFO             PET3 ESMF_COMM                   : openmpi
20210420 195134.039 INFO             PET3 ESMF_MOAB                   : enabled
20210420 195134.039 INFO             PET3 ESMF_LAPACK                 : enabled
20210420 195134.039 INFO             PET3 ESMF_NETCDF                 : enabled
20210420 195134.039 INFO             PET3 ESMF_PNETCDF                : disabled
20210420 195134.039 INFO             PET3 ESMF_PIO                    : enabled
20210420 195134.039 INFO             PET3 ESMF_YAMLCPP                : enabled
20210420 195134.039 INFO             PET3  Beginning Test, file ESMF_VMAllToAllUTest.F90, line 72
20210420 195134.039 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210420 195134.039 INFO             PET3  PASS  AllToAll Test iarray1 -> iarray2, ESMF_VMAllToAllUTest.F90, line 127
20210420 195134.039 INFO             PET3  PASS  Verify iarray2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 141
20210420 195134.039 INFO             PET3  PASS  AllToAll Test iarray2 -> iarray3, ESMF_VMAllToAllUTest.F90, line 151
20210420 195134.039 INFO             PET3  PASS  Verify iarray3 data against iarray1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 165
20210420 195134.039 INFO             PET3  PASS  AllToAll Test r4array1 -> r4array2, ESMF_VMAllToAllUTest.F90, line 177
20210420 195134.039 INFO             PET3  PASS  Verify r4array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 191
20210420 195134.039 INFO             PET3  PASS  AllToAll Test r4array2 -> r4array3, ESMF_VMAllToAllUTest.F90, line 201
20210420 195134.039 INFO             PET3  PASS  Verify r4array3 data against r4array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 215
20210420 195134.039 INFO             PET3  PASS  AllToAll Test r8array1 -> r8array2, ESMF_VMAllToAllUTest.F90, line 227
20210420 195134.039 INFO             PET3  PASS  Verify r8array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 241
20210420 195134.039 INFO             PET3  PASS  AllToAll Test r8array2 -> r8array3, ESMF_VMAllToAllUTest.F90, line 251
20210420 195134.039 INFO             PET3  PASS  Verify r8array3 data against r8array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 265
20210420 195134.039 INFO             PET3  Ending Test, file ESMF_VMAllToAllUTest.F90, line 267
20210420 195134.039 INFO             PET3 Finalizing ESMF
