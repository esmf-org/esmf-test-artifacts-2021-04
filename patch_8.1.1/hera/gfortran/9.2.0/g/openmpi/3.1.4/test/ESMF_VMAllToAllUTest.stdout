build time -- 2021-04-27 06:48:20
[1619506580.988437] [h16c15:287189:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1619506580.992867] [h16c15:287189:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1619506580.988438] [h16c15:287191:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1619506580.992920] [h16c15:287191:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1619506580.988792] [h16c15:287192:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1619506580.993239] [h16c15:287192:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1619506580.988747] [h16c15:287190:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1619506580.993180] [h16c15:287190:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
  Beginning Test, file ESMF_VMAllToAllUTest.F90, line 72
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_VMAllToAllUTest.F90, line 72
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_VMAllToAllUTest.F90, line 72
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_VMAllToAllUTest.F90, line 72
  NUMBER_OF_PROCESSORS           4
  PASS  AllToAll Test iarray1 -> iarray2, ESMF_VMAllToAllUTest.F90, line 127
  PASS  AllToAll Test iarray1 -> iarray2, ESMF_VMAllToAllUTest.F90, line 127
  PASS  AllToAll Test iarray1 -> iarray2, ESMF_VMAllToAllUTest.F90, line 127
  PASS  Verify iarray2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 141
  PASS  AllToAll Test iarray1 -> iarray2, ESMF_VMAllToAllUTest.F90, line 127
  PASS  Verify iarray2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 141
  PASS  Verify iarray2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 141
  PASS  Verify iarray2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 141
  PASS  AllToAll Test iarray2 -> iarray3, ESMF_VMAllToAllUTest.F90, line 151
  PASS  AllToAll Test iarray2 -> iarray3, ESMF_VMAllToAllUTest.F90, line 151
  PASS  Verify iarray3 data against iarray1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 165
  PASS  AllToAll Test iarray2 -> iarray3, ESMF_VMAllToAllUTest.F90, line 151
  PASS  Verify iarray3 data against iarray1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 165
  PASS  AllToAll Test iarray2 -> iarray3, ESMF_VMAllToAllUTest.F90, line 151
  PASS  Verify iarray3 data against iarray1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 165
  PASS  Verify iarray3 data against iarray1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 165
  PASS  AllToAll Test r4array1 -> r4array2, ESMF_VMAllToAllUTest.F90, line 177
  PASS  AllToAll Test r4array1 -> r4array2, ESMF_VMAllToAllUTest.F90, line 177
  PASS  AllToAll Test r4array1 -> r4array2, ESMF_VMAllToAllUTest.F90, line 177
  PASS  Verify r4array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 191
  PASS  Verify r4array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 191
  PASS  AllToAll Test r4array1 -> r4array2, ESMF_VMAllToAllUTest.F90, line 177
  PASS  Verify r4array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 191
  PASS  Verify r4array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 191
  PASS  AllToAll Test r4array2 -> r4array3, ESMF_VMAllToAllUTest.F90, line 201
  PASS  AllToAll Test r4array2 -> r4array3, ESMF_VMAllToAllUTest.F90, line 201
  PASS  Verify r4array3 data against r4array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 215
  PASS  AllToAll Test r4array2 -> r4array3, ESMF_VMAllToAllUTest.F90, line 201
  PASS  Verify r4array3 data against r4array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 215
  PASS  AllToAll Test r4array2 -> r4array3, ESMF_VMAllToAllUTest.F90, line 201
  PASS  Verify r4array3 data against r4array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 215
  PASS  Verify r4array3 data against r4array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 215
  PASS  AllToAll Test r8array1 -> r8array2, ESMF_VMAllToAllUTest.F90, line 227
  PASS  AllToAll Test r8array1 -> r8array2, ESMF_VMAllToAllUTest.F90, line 227
  PASS  AllToAll Test r8array1 -> r8array2, ESMF_VMAllToAllUTest.F90, line 227
  PASS  Verify r8array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 241
  PASS  AllToAll Test r8array1 -> r8array2, ESMF_VMAllToAllUTest.F90, line 227
  PASS  Verify r8array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 241
  PASS  Verify r8array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 241
  PASS  AllToAll Test r8array2 -> r8array3, ESMF_VMAllToAllUTest.F90, line 251
  PASS  Verify r8array2 data after alltoall, ESMF_VMAllToAllUTest.F90, line 241
  PASS  AllToAll Test r8array2 -> r8array3, ESMF_VMAllToAllUTest.F90, line 251
  PASS  AllToAll Test r8array2 -> r8array3, ESMF_VMAllToAllUTest.F90, line 251
  PASS  Verify r8array3 data against r8array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 265
  Ending Test, file ESMF_VMAllToAllUTest.F90, line 267
  PASS  Verify r8array3 data against r8array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 265
  Ending Test, file ESMF_VMAllToAllUTest.F90, line 267
  PASS  Verify r8array3 data against r8array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 265
  Ending Test, file ESMF_VMAllToAllUTest.F90, line 267
  PASS  AllToAll Test r8array2 -> r8array3, ESMF_VMAllToAllUTest.F90, line 251
  PASS  Verify r8array3 data against r8array1 after alltoallv, ESMF_VMAllToAllUTest.F90, line 265
  Ending Test, file ESMF_VMAllToAllUTest.F90, line 267
  PET           1  Test Elapsed Time    17.395999999999965       msec.
  PET           3  Test Elapsed Time    17.198999999999963       msec.
  PET           0  Test Elapsed Time    17.372000000000000       msec.
  PET           2  Test Elapsed Time    17.297000000000008       msec.
